### PR TITLE
Horrific Journeys integration

### PR DESCRIPTION
**Pull request by redwolf2**
https://github.com/NPBruce/valkyrie/issues/868

This is only here to track the progress. I will notify you, when it is ready.

### CODE REVIEW

File: unity/Assets/StreamingAssets/content/MoM/hj/monsters.ini [hunk 1]
```diff
@@ -0,0 +1,39 @@
+[MonsterWarlock]
+name={ffg:MONSTER_WARLOCK}
+image="{import}/img/Monster_Warlock"
+traits=humanoid
+awareness=4
+horror=3
+brawn=2
+health=4
+healthperhero=1
```
Comment (redwolf2) [inline unity/Assets/StreamingAssets/content/MoM/hj/monsters.ini]: I an not sure about the monster health and how healthperhero realy works. Does health per hero means for 1 more hero the health is raised by 1 or does it mean that if healthperhero=2, then every two heroes the health is raised by one.
The app really shows different results. I thought I have seen a DimensionalShambler with a health of 6 using two investigators, which raised to 8 when using three.

**Comment by redwolf2** [comment]
All good to go. I am still not sure about the Formless Spawn health, though.

File: unity/Assets/StreamingAssets/content/MoM/hj/monsters.ini (hunk 1 above)
Comment (redwolf2) [reply unity/Assets/StreamingAssets/content/MoM/hj/monsters.ini]: Formless Spawn does spawn in What Lies Within, but this is a special, non regular monster.

Comment (jlederer2) [reply unity/Assets/StreamingAssets/content/MoM/hj/monsters.ini]: I think that you have health and healthperhero correct.  Dimensional Shambler starts with health of 6 (two investigators is the minimum allowed) and adds two per hero so health with three heroes = 8.

**Pull request merged**